web/src/locales/de.json

```diff
@@ -94,9 +94,8 @@
   "estimation-card": {
     "link": "Lesen Sie mehr über unsere statistischen Schätzmodelle",
     "estimated_time_slicer_average": {
-      "title": "Die Daten werden derzeit geschätzt",
-      "pill": "Nicht verfügbar",
-      "body": "Unsere Datenquellen stellen aktuell keine Werte für diese Zone zur Verfügung. Die angezeigten Werte sind Schätzungen, die auf historischen Messungen basieren und durch verifizierte Daten ersetzt werden, sobald diese verfügbar sind. Bitte beachten Sie, dass geschätzte Werte nicht immer genau sind, da sie aktuelle Wetterereignisse nicht berücksichtigen."
+      "title": "Die Daten sind vorläufig",
```

Comment:
```suggestion
      "title": "Vorläufige Daten",
```

Reply:
In the English one we used "Data is preliminary" instead of "Preliminary data", dunno if there's a good reason - what do you think @Alportan?

Reply:
It just sounded a bit more natural to me this way round in German :) but we could also stick with "Die Daten sind vorläufig" or "Die angezeigten Daten sind vorläufig".

Reply:
The only reason was to keep it consistent with the estimates, where we say "Data is estimated" + reason in the pill (incomplete, unavailable etc). I think we can keep it this way in German, shouldn't be an issue, especially that there's no pill here 🙌

Reply:
Okay thanks, I also just noticed we do the same for other estimation titles in German :)